### ISSUE TITLE
indexedDB in Options - latest version

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alignment-editor-new#638]
related to #634 
1) 'Set all to advanced' does turn on all the options listed, except for tokenizer and indexedDB, which is already on by default.  I found weird that all options are checked (for a good reason as they are set on), except for indexedDB that is already on from the beginning.  I would rather move the tokenizer and the indexedDB in a separate block, still in Advanced and say something like these options are already the default ... if you want to change something you can do it here...

2) when I turned off all Options, including indexedDB, the next time I click on Set all to advanced, all Options are checked except for indexedDB. This is a contradiction with the initial set of default Options, specifically the advanced options.

![Screen Shot 2022-02-01 at 5 08 59 PM](https://user-images.githubusercontent.com/41396793/152005515-c7f59636-4e55-4901-84ae-1ee1bf3d5379.png)

**Comment by irina060981** [comment]
Let's go by steps:

1. We have default option's set and we have  a but that set's option's to advanced state.
2. When we click reset options - they return to their default state (it is defined for each of them)
3. When we click to advanced - we set them advanced values (for those options that has advanced value according to the PDF with requirements)
4. Add IndexedDB support and Tokenizer are not options that are dependent on type of a user (standard or advanced). They are dependent on technical environment - does a user has not turned off IndexedDB in the browser and does he has Internet connection for using remote tokenizer. That's why these options sre not changed according to advanced state. But setting to default - means that all option's changes are deleted from history and thy are returned to initial state. That's why in these case these options could be changes.

So from my point of view this workflow suits requirements. 
If you want to change them - let's fix changes in this issue.

**Comment by monzug** [comment]
Right now, it's confusing. Set all to advanced implies to me that ALL options are set. As you said, tokenizer and indexedDB are the exception to this behavior.  As I got tricked by the uncheck button for indexedDB when I clicked on Set all to advanced, it means that it's NOT straightforward.  said so, I would suggest to move tokenizer and indexedDB below the Set all to advanced button. In this way it should be obvious that they do not follow the other Options.
